.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+### What's being changed:
+
+
+### Review checklist
+
+- [ ] Documentation has been updated, if necessary. Link to changed documentation:
+- [ ] Chaos pipeline run or not necessary. Link to pipeline:
+- [ ] All new code is covered by tests.
+- [ ] Performance tests have been run or not necessary.
```

Comment:
I would treat the whole list as optional, so that we would leave only the relevant points for our PR's that we want to run. So I would leave this point in the template, but I would agree that we can remove those points which are not relevant to given change. I would treat this list only as a reminder of things that one can perform.